CHANGELOG.md

```diff
@@ -1,5 +1,75 @@
 # Changelog
 
+## [1.27.0] - 2024-01-29
+
+This long-awaited Rustup release has gathered all the new features and fixes since April 2023.
+These changes have greatly improved Rustup's maintainability, user experience, compatibility and documentation quality.
```

Comment:
I feel like this is overstating things a little? I don't feel like the Added/Changed lines below represent "great" improvements.

Reply:
@djc I guess you can directly edit the changelog on the top of this thread and I'll reflect the changes in the actual file, how does that sound?